shared/src/models/radiogram/resource-request-radiogram.ts

```diff
@@ -26,6 +32,10 @@ export class ResourceRequestRadiogram implements Radiogram {
     @ValidateNested()
     readonly status: ExerciseRadiogramStatus;
 
+    @IsOptional()
+    @IsBoolean()
+    readonly resourcesPromised?: boolean;
```

Comment:
Could you please add a migration for this property to ensure that this key is present on the class and not implicitly set to `undefined` since it is missing.

Reply:
And keep in mind to add test scenarios